### PR TITLE
Feature/brand pivot hero block

### PR DESCRIPTION
**Pull request by gustaveen**
## New hero block

hero

### CODE REVIEW

File: src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  image: '',
+  image_mobile: '',
+}
+
+const image = 'https://cdn.hedvig.com/www/referrals/referrals-clean.png'
+
+export const Default = () => (
+  <HeroImageBlockBrandPivot
+    {...heroProps}
+    color={select('color', colors, colors.standardInverse)}
```
Comment (gustaveen) [inline src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx]: Having a hard time figuring out how to pass in color options to `addon-knobs` without getting type errors. It basically needs a string and not and object. Do you have any suggestion @palmenhq

File: src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  gray700: {
+    _uid: '6ecde11d-ba0a-48fb-9b7b-e6dbf31415d9',
+    color: 'gray700',
+    plugin: 'hedvig_minimal_color_picker',
+  },
+  gray500Inverse: {
+    _uid: '6ecde11d-ba0a-48fb-9b7b-e6dbf31415d9',
+    color: 'gray500-inverse',
+    plugin: 'hedvig_minimal_color_picker',
+  },
```
Comment (palmenhq) [inline src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx]: these'll be used in more places no? should we extract it to somewhere where its reusable?

File: storyblok/components.json [hunk 3]
```diff
@@ -55,7 +55,9 @@
           "type": "bloks",
```
Comment (palmenhq) [inline storyblok/components.json]: we should run prettier on this file :)

**Review by palmenhq — COMMENTED** [review]
wooh!

File: src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  image: '',
+  image_mobile: '',
+}
+
+const image = 'https://cdn.hedvig.com/www/referrals/referrals-clean.png'
+
+export const Default = () => (
+  <HeroImageBlockBrandPivot
+    {...heroProps}
+    color={select('color', colors as any, colors.standardInverse as any)}
```
Comment (palmenhq) [inline src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx]: maybe we could instead do something like
```ts
color={
colors[select('color', Object.keys(colors), 'standard-inverse')]
}
```

Comment (gustaveen) [reply src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx]: Yes that's what I wanted too 🙂 But it needs a the `MinimalColorComponent` object as an option

File: storyblok/components.json (hunk 3 above)
Comment (gustaveen) [reply storyblok/components.json]: I saw that i messed up with this file and commited removed blocks, will fix 🙈

File: src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx (hunk 2 above)
Comment (gustaveen) [reply src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx]: that might be good! what do you think about some utils for storybook?

File: src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx (hunk 4 above)
Comment (palmenhq) [reply src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx]: Yes, isn't that what this'd do? Or not really?

File: src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx (hunk 2 above)
Comment (gustaveen) [reply src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx]: I've created a `utils/storybook.ts` and moved it there.

**Comment by gustaveen** [comment]
Updated now @palmenhq Thanks a lot for the typescript support ❤️

File: src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.tsx [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+  headline,
+  text,
+  text_color,
+  image,
+  index,
+  image_mobile,
+  use_display_font = false,
+  show_hedvig_wordmark,
+}) => {
+  const isTablet = useMediaQuery({ maxWidth: 801 })
```
Comment (palmenhq) [inline src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.tsx]: Does this work though? Considering we're using SSR. I think it'd be better to use client-safe css media queries

**Review by palmenhq — COMMENTED** [review]
Amaze! Just one thought

File: src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.tsx (hunk 5 above)
Comment (gustaveen) [reply src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.tsx]: Ah, good catch, that might be a problem. Not sure how this works under the hood but if would it work if you use the component syntax like we do in other places?:
```suggestion
<MediaQuery query="(max-width: 801px)"> 
   {...}
</MediaQuery>
```

Comment (gustaveen) [reply src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.tsx]: Let's just stick to good ol' css https://github.com/HedvigInsurance/web-next/pull/387/commits/0c8fbe335f42e702c832af5e93804aa5008d8834 😄

**Review by palmenhq — APPROVED** [review]
amaze!

**Pull request merged**